main/adv_post.c

```diff
@@ -126,8 +141,20 @@ parse_adv_report_from_uart(const re_ca_uart_payload_t *const msg, adv_report_t *
     return err;
 }
 
-void
-adv_post_send(void *arg)
+static void
```

Comment:
Please add a comment describing when this function gets called and what *arg is expected to contain.